src/documents/ComponentConfigurationOptions.md

```diff
@@ -260,59 +247,62 @@ See also: [Component Grades](ComponentGrades.md)
   </tr>
   <tr>
     <th>See also</th>
-    <td><a href="FrameworkConcepts.md#model-objects">Model Objects</a><br/>
-        <a href="ChangeApplierAPI.md">ChangeApplier API</a><br/>
-        <code><a href="#applier">applier</a></code><br/>
-        <code><a href="#changeapplieroptions">changeApplierOptions</a></code></td>
+    <td><a href="OptionsMerging.md">Options Merging</a></td>
   </tr>
 </table>
 
-### applier ###
+### `dynamicComponents` ###
 <table>
   <tr>
     <th>Description</th>
-    <td>A <a href="ChangeApplier.md">ChangeApplier</a> object for the model provided with the <code><a href="#model">model</a></code> option.</td>
+    <td>An object containing named definitions of the component's dynamic subcomponents. Rather than exactly one subcomponent being associated with its parent from these records, there may be one subcomponent per element of an array, or one per firing of an event.</td>
   </tr>
   <tr>
     <th>Notes</th>
-    <td>It is not necessary to provide an applier: By default, an applier will be created with <code>fluid.makeChangeApplier()</code>, using any options specified with <code><a href="#changeapplieroptions">changeApplierOptions</a></code>.
-
-This option is most commonly used to share a common ChangeApplier between components in a component tree: the <code>applier</code> option can be used to reference the ChangeApplier of another component in the tree.</td>
+    <td>Some special context names may be available within the subcomponent's definition block, for example <code>{source}</code> and <code>{sourcePath}</code> or <code>{arguments}</code>. <em>This framework facility will be replaced by a more declarative equivalent in time and should be used with caution.</em></td>
```

Comment:
I think I'd go ahead and say exactly this. "If you think you might need to use this feature, ask on < link to community forums > for advice" or something to that effect.